UDx-container/README.md

```diff
@@ -114,13 +114,7 @@ This repository provides `vsdk-*` scripts to help you test and compile your UDx
 | `vsdk-g++` | Executes the g++ compiler in the UDx container. |
 | `vsdk-make` | Executes `make` in the current working directory in the UDx container. This allows you to develop UDxs locally and compile them with the tools available in the UDx container. |
 
-You will likely use `vsdk-make` the most often. It behaves exactly like `make`, but it compiles your files in the development environment mounted in the UDx container.
-
-These scripts use the contents of `/etc/os-release` to determine whether the container has a `centos` or `ubuntu` tag. If your host uses a different distribution than your development environment, you can edit `vsdk-bash` directly to change the default setting. Alternatively, you can change the default interactively by defining the OSTAG [environment variable](#environment-variables) when you execute `vsdk-make`:
-
-```shell
-$ OSTAG=ubuntu vsdk-make
-```
+These scripts use the contents of `/etc/os-release` to determine whether the container has a `centos` or `ubuntu` tag. If your host uses a different distribution than your development environment, you can edit `vsdk-bash` directly to change the default setting. Alternatively, you can change the default interactively by defining the `OSTAG` [environment variable](#environment-variables) when you execute `vsdk-make`. For details,see [Compile UDxs](#compile-udxs).
```

Comment:
Can we leave the part out about editing vsdk-bash?  They could certainly do that, but it's better to only suggest using OSTAG, imho.  @dmankins

Reply:
I don't know that I want to encourage people to set OSTAG permanently (which is why the example talks about "OSTAG=... vsdk-make ..."

It's a git repo, I see no problem with suggesting ways people can customize their scripts.

Hmm.  I don't typically use aliases, but maybe this is a good place to suggest "alias vsdk-make="OSTAG=<whatever> path-to-vsdk-make"?